### PR TITLE
Trad/overview

### PR DESCRIPTION
**Pull request by fdechelle**
Translated modules/generalites
overview/templates.rst needs some checks.

### CODE REVIEW

**Comment by cconard96** [comment]
Build is visible at: https://glpi-user-documentation--44.org.readthedocs.build/fr/45

File: source/modules/overview/templates.rst [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+A template is attached to the entity where it has been created and can be visible in sub-entities.
+
+Default templates can be defined at entity level or at profile level. For the profiles, only the template of the root entity that are visible from sub-entities can be attached.
+
+Default templates can also be defined by :ref:`ticket category <tickets-categories>`.
+
+When creating a ticket, the used template is, with priority order:
+
+1. First, the template defined in the defined category for the defined type
```
Comment (trasher) [inline source/modules/overview/templates.rst:75]:
```suggestion
1. First, the template defined in the selected category and type
```

File: source/modules/overview/actions.rst [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+#. Connect to GLPI ;
+#. Go to the object page, for example, to display a computer, go to `Assets > Computers` ;
+#. Or alternatively: search the object in the search engine available on top of objects list ;
+#. Click on object name..
+
+
+Attaching a document
+^^^^^^^^^^^^^^^^^^^^
+
+For all inventory objects, attaching a document is realized in the same way:
```
Comment (trasher) [inline source/modules/overview/actions.rst:70]: Not only inventory objects as far as I know

File: source/modules/overview/templates.rst [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+---------------------------
+
+Introduction
+^^^^^^^^^^^^
+
+For some object types in GLPI, it is possible to create new objects (computers for example) using predefined templates. A template defines a creation model in which some fields are either already fixed or computed using a function, for example in the case of a computer a function which would compute the inventory number.
+
+Template management is accessible via the button |btn_template| located in the menu bar.
+
+Inventory objects
```
Comment (trasher) [inline source/modules/overview/templates.rst:11]: `s/objects/assets/` maybe?

File: source/modules/overview/actions.rst (hunk 2 above)
Comment (fdechelle) [reply source/modules/overview/actions.rst:70]: Then "for all objects"?

File: source/modules/overview/templates.rst (hunk 3 above)
Comment (fdechelle) [reply source/modules/overview/templates.rst:11]: agree

File: source/modules/overview/actions.rst (hunk 2 above)
Comment (cconard96) [reply source/modules/overview/actions.rst:70]: There are a few non-inventory types that can have Documents attached.
The current list of document types:
```php
$CFG_GLPI["document_types"] = ['Budget', 'CartridgeItem', 'Change', 'Computer',
                                                  'ConsumableItem', 'Contact', 'Contract',
                                                  'Document', 'Entity', 'KnowbaseItem', 'Monitor',
                                                  'NetworkEquipment', 'Peripheral', 'Phone',
                                                  'Printer', 'Problem', 'Project', 'ProjectTask',
                                                  'Reminder', 'Software', 'Line',
                                                  'SoftwareLicense', 'Supplier', 'Ticket','User',
                                                  'Certificate', 'Cluster', 'ITILFollowup', 'ITILSolution',
                                                   'ChangeTask', 'ProblemTask', 'TicketTask', 'Appliance'];
```

Comment (fdechelle) [reply source/modules/overview/actions.rst:70]: I  propose:
"When available for the object type, attaching a document is realized...."

File: source/modules/overview/templates.rst [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+* can contain the following special characters which are replaced automatically:
+
+   * ``\g`` lookup of number in all identical fields based on the same format;
+   * ``#``: is replaced by a counter having as many digits as there are consecutive ``#`` characters;
+   * ``\Y``:  is replaced by the year on 4 digits;
+   * ``\y``:  is replaced by the year on 2 digits;
+   * ``\m``: is replaced by the month;
+   * ``\d``: is replaced by the day.
+
+.. ??? don't understand ``\g`` recherche du numéro parmi tous les champs identiques basés sur le même format ;
```
Comment (trasher) [inline source/modules/overview/templates.rst:41]: Do not understand neither... @orthagh?

File: source/modules/overview/templates.rst [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+For example, when creating a printer, it is possible in the printer template to use this mechanism to generate automatically inventory numbers.
+
+Let suppose that we want an inventory number under the form: `YEAR+MONTH+DAY+fixed code structure equals to 555 + fixed operation code equals to 1234 + 2_DIGITS_COUNTER`; then the inventory number field in the template will be: ``<\Y-\m-\d-555-1234-##\>``.
+
+On each printer creation, the first printer will have its inventory number be `1984-JAN-02-555-1234-01`, the second printer `1984-JAN-02-555-1234-02` and so on.
+
+.. ??? please check example
```
Comment (trasher) [inline source/modules/overview/templates.rst:49]: Seems valid; dunno if this is a "real world" example

File: source/modules/overview/actions.rst (hunk 2 above)
Comment (trasher) [reply source/modules/overview/actions.rst:70]: s/object type/item type/

Comment (fdechelle) [reply source/modules/overview/actions.rst:70]: Agree

File: source/modules/overview/templates.rst (hunk 3 above)
Comment (fdechelle) [reply source/modules/overview/templates.rst]: and changed

File: source/modules/overview/actions.rst (hunk 2 above)
Comment (fdechelle) [reply source/modules/overview/actions.rst]: Changed

File: source/modules/overview/templates.rst [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+A template is attached to the entity where it has been created and can be visible in sub-entities.
+
+Default templates can be defined at entity level or at profile level. For the profiles, only the template of the root entity that are visible from sub-entities can be attached.
+
+Default templates can also be defined by :ref:`ticket category <tickets-categories>`.
+
+When creating a ticket, the used template is, with priority order:
+
+1. First, the template defined in the selected category and type
+.. ??? 1. ouille  Celui défini dans la catégorie définie pour le type défini
```
Comment (trasher) [inline source/modules/overview/templates.rst:76]:
```suggestion
```

File: source/modules/overview/templates.rst (hunk 5 above)
Comment (trasher) [reply source/modules/overview/templates.rst:49]:
```suggestion
```

File: source/modules/overview/templates.rst (hunk 4 above)
Comment (trasher) [reply source/modules/overview/templates.rst:41]: Translation seems correct anyways
```suggestion
```

**Review by trasher — APPROVED** [review]

File: source/modules/overview/actions.rst [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+   #. Select the checkbox located on the left of the object name ;
+   #. Click on button **Actions** and choose **Add a Document**
+
+GLPI will then display the object form with newly attached document.
+
+
+Attaching a contract
+^^^^^^^^^^^^^^^^^^^^
+
+For all inventory objects, attaching a contract (if a contract can be attached to the object) is realized in the same way:
```
Comment (cconard96) [inline source/modules/overview/actions.rst:96]: Applies to more than just inventory types. Same as documents.

File: source/modules/overview/actions.rst [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+   #. Select tab **Contracts** ;
+   #. Select a contract ;
+   #. Click on button **Add**.
+
+- Mass attachment :
+
+   #. Select the checkbox located on the left of the object name ;
+   #. Click on button **Actions** and choose **Add a Contract**
+
+GLPI will then display the object form with newly attached document.
```
Comment (cconard96) [inline source/modules/overview/actions.rst:116]:
```suggestion
GLPI will then display the object form with newly attached contract.
```

**Review by cconard96 — CHANGES_REQUESTED** [review]

File: source/modules/overview/actions.rst (hunk 7 above)
Comment (fdechelle) [reply source/modules/overview/actions.rst:96]: I propose to change it to (similar to documents):

_When available for the item type, attaching a contract is realized in the same way:_

File: source/modules/overview/templates.rst [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+
+For example, when creating a printer, it is possible in the printer template to use this mechanism to generate automatically inventory numbers.
+
+Let suppose that we want an inventory number under the form: `YEAR+MONTH+DAY+fixed code structure equals to 555 + fixed operation code equals to 1234 + 2_DIGITS_COUNTER`; then the inventory number field in the template will be: ``<\Y-\m-\d-555-1234-##\>``.
+
+On each printer creation, the first printer will have its inventory number be `1984-JAN-02-555-1234-01`, the second printer `1984-JAN-02-555-1234-02` and so on.
+
+
+
+Ticket templates
```
[the remark this answers is not in the document]
Comment (fdechelle) [reply source/modules/overview/templates.rst:50]: Should it be changed now? It means changing (or rewriting) the entire paragraph.
My suggestion:

- keep it for the PR
- add an issue in glpi-project/doc to update it

File: source/modules/overview/actions.rst (hunk 7 above)
Comment (fdechelle) [reply source/modules/overview/actions.rst]: Changed

File: source/modules/overview/templates.rst (hunk 9 above)
[the remark this answers is not in the document]
Comment (cconard96) [reply source/modules/overview/templates.rst:50]: I don't think it needs to be changed here. I've changed my review to approved.

**Review by cconard96 — APPROVED** [review]

**Pull request merged**